src/components/claims/claim-details/components/MemberInformation.tsx

```diff
@@ -68,97 +84,107 @@ const MemberInformation: React.FC<{
       label: `Go to member`,
       keys: [Keys.Option, Keys.M],
       onResolve: () => {
-        history.push(`/members/${member.memberId}`)
+        history.push(`/members/${member?.memberId}`)
```

Comment:
Let's use `memberId` input parameter instead 🐶 
```suggestion
        history.push(`/members/${memberId}`)
```